src/ui/public/agg_types/buckets/range.js

```diff
@@ -1,6 +1,6 @@
 import { AggTypesBucketsBucketAggTypeProvider } from 'ui/agg_types/buckets/_bucket_agg_type';
 import { AggTypesBucketsCreateFilterRangeProvider } from 'ui/agg_types/buckets/create_filter/range';
-import { FieldFormat } from 'ui/index_patterns/_field_format/field_format';
+import { FieldFormat } from '../../../../core_plugins/kibana/common/field_formats/field_format';
```

Comment:
Should we be reaching inside the `core_plugins/kibana` folder from within `src/ui/public`? What made you and @spalger choose to put the `common` folder inside the `kibana` plugin and not a direct descendant of `src/`? This same issue applies to a few other situations below as well.